test/test_artifacts/v2/scripts/run_maxdome_jupyter_server_extension_test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pip install pytest-jupyter
```

Comment:
can you move this to the corresponding docker file and use micromamba for installation? Seems like this package is availabe in conda forge

https://anaconda.org/conda-forge/pytest-jupyter

Note: Use --freeze-installed flag when installing from conda forge

Reply:
Fixed with follow up commit.